esphome/components/scd30/sensor.py

```diff
@@ -81,6 +89,9 @@ async def to_code(config):
     if CONF_TEMPERATURE_OFFSET in config:
         cg.add(var.set_temperature_offset(config[CONF_TEMPERATURE_OFFSET]))
 
+    if CONF_UPDATE_INTERVAL in config:
+        cg.add(var.set_update_interval(config[CONF_UPDATE_INTERVAL]))
```

Comment:
You have a default in the schema so no need to check `if` it is there. 
```suggestion
    cg.add(var.set_update_interval(config[CONF_UPDATE_INTERVAL]))
```